devito/operator/operator.py

```diff
@@ -148,46 +157,32 @@ def _build(cls, expressions, **kwargs):
         if any(not isinstance(i, Eq) for i in expressions):
             raise InvalidOperator("Only `devito.Eq` expressions are allowed.")
 
-        name = kwargs.get("name", "Kernel")
-        dse = kwargs.get("dse", configuration['dse'])
-
         # Python-level (i.e., compile time) and C-level (i.e., run time) performance
         profiler = create_profile('timers')
 
-        # Lower input expressions to internal expressions (e.g., attaching metadata)
+        # Lower input expressions
         expressions = cls._lower_exprs(expressions, **kwargs)
 
-        # Group expressions based on their iteration space and data dependences
-        # Several optimizations are applied (fusion, lifting, flop reduction via DSE, ...)
-        clusters = clusterize(expressions, dse_mode=set_dse_mode(dse))
+        # Group expressions based on iteration spaces and data dependences
+        clusters = cls._lower_clusters(expressions, **kwargs)
 
-        # Lower Clusters to a Schedule tree
-        stree = st_build(clusters)
+        # Turn Clusters into a Schedule tree
+        stree = cls._lower_stree(clusters, **kwargs)
```

Comment:
I am a bit confused here. Lower Clusters is changed to Turn into... 
However the new name of the function is lower_stree ?